hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/config/GrpcConfiguration.java

```diff
@@ -40,22 +41,28 @@ class GrpcConfiguration {
     @Bean
     GrpcServerConfigurer grpcServerConfigurer(GrpcProperties grpcProperties) {
         NettyProperties nettyProperties = grpcProperties.getNetty();
-        Executor executor = new ThreadPoolExecutor(
-                nettyProperties.getExecutorCoreThreadCount(),
-                nettyProperties.getExecutorMaxThreadCount(),
-                nettyProperties.getThreadKeepAliveTime().toSeconds(),
-                TimeUnit.SECONDS,
-                new SynchronousQueue<>(),
-                new ThreadFactoryBuilder()
-                        .setDaemon(true)
-                        .setNameFormat("grpc-executor-%d")
-                        .build());
-
-        return serverBuilder -> ((NettyServerBuilder) serverBuilder)
-                .executor(executor)
-                .maxConnectionIdle(nettyProperties.getMaxConnectionIdle().toSeconds(), TimeUnit.SECONDS)
-                .maxConcurrentCallsPerConnection(nettyProperties.getMaxConcurrentCallsPerConnection())
-                .maxInboundMessageSize(nettyProperties.getMaxInboundMessageSize())
-                .maxInboundMetadataSize(nettyProperties.getMaxInboundMetadataSize());
+        return serverBuilder -> getServerBuilder(serverBuilder, nettyProperties);
+    }
+
+    private ServerBuilder<?>  getServerBuilder(ServerBuilder<?> serverBuilder, NettyProperties nettyProperties) {
```

Comment:
`GrpcServerConfigurer` extends `Consumer` which should return no result, so in case you prefer a new private method, change it to
```suggestion
    private void customizeServerBuilder(ServerBuilder<?> serverBuilder, NettyProperties nettyProperties) {
```